Contributing.md

```diff
@@ -110,7 +110,7 @@ You can run these tests yourself with
 `nox -s clippy-all`
 
 #### Tests
-`cargo test --features full`
+`nox -s test`
```

Comment:
Is it potentially worth mentioning both options here? I tend to use `cargo test` to selectively run the ones I care about (and it's much quicker), though I agree most contributors would benefit from having this recommendation.

Reply:
There's also `nox -s test-rust` :)

Reply:
But yes, for running a specific test `cargo test` is probably still best, I'll add a sentence.

Oh, and for adding a hint how to run Rust tests, is there a way to run a specific test from `pytests`?

Reply:
I think `cd pytests && nox -s test -- -k <test>` (the `-k` switch goes to underlying `pytest` execution)

Reply:
Or `nox -f pytests/noxfile.py -s test -- -k test` to avoid the `cd`.